src/Tribe/Customizer/Text.php

```diff
@@ -0,0 +1,272 @@
+					color: rgba( ' . $primary_rgb . ', 0.62 );
+				}
+			';
+
+		}
+
+		if ( $customizer->has_option( $this->ID, 'secondary_text_color' ) ) {
+			$template .= '
+				.tribe-common .tribe-common-form-control-toggle__label,
+				.tribe-events .tribe-events-calendar-list__event-date-tag-weekday {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border, 
+				.tribe-common a.tribe-common-c-btn-border {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-form-control-text__input::placeholder {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-common--breakpoint-medium.tribe-events .tribe-events-c-nav__prev, 
+				.tribe-common--breakpoint-medium.tribe-events .tribe-events-c-nav__next {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+
+			// PRO Styles.
+			$template .= '
+				.tribe-events .tribe-events-calendar-month__calendar-event-datetime,
+				.tribe-events-pro .tribe-events-pro-photo__event-date-tag-month,
+				.tribe-events-pro .tribe-events-pro-week-grid__event-datetime,
+				.tribe-events-pro .tribe-events-pro-map__event-date-tag-month,
+				.tribe-common--breakpoint-medium.tribe-events-pro .tribe-events-pro-map__event-distance {
+					color: <%= text.secondary_text_color %>;
+				}
+			';
+			
+		}
+
+		return $template;
+	}
+
+	/**
+	 * Setup the Customizer section.
+	 * 
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function setup() {
+		$this->defaults = [
+			'primary_text_color'   => '#141827',
+			'secondary_text_color' => '#727272',
+		];
+
+		$this->arguments = [
+			'priority'    => 60,
+			'capability'  => 'edit_theme_options',
+			'title'       => esc_html__( 'Text', 'the-events-calendar' ),
+			'description' => esc_html__( 'Options selected here will override what was selected in the General section.', 'the-events-calendar' ),
+		];
+	}
+
+	/**
+	 * Create the Fields/Settings for this sections.
+	 *
+	 * @param  WP_Customize_Section $section The WordPress section instance.
+	 * @param  WP_Customize_Manager $manager [description]
+	 *
+	 * @return void
+	 */
+	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
+		$customizer = Tribe__Customizer::instance();
+
+		$manager->add_setting(
+			$customizer->get_setting_name( 'primary_text_color', $section ),
+			[
+				'default'              => $this->get_default( 'primary_text_color' ),
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new WP_Customize_Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'primary_text_color', $section ),
+				[
+					'label'   => esc_html__( 'Primary Text Color', 'the-events-calendar' ),
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 1c200cfdbcc0e75885e7cab9b52ac41f</sub>